compiler/src/dotty/tools/dotc/core/Symbols.scala

```diff
@@ -91,10 +91,24 @@ object Symbols {
 
     private[core] def invalidateDenotCache(): Unit = { checkedPeriod = Nowhere }
 
+    private[core] var cachedInfoOrCompleter: Type = _
+    private[core] var cachedFlags: FlagSet = _
+    private[core] var cachedName: Name = _
```

Comment:
It's minor, but these fields will appear after the `val id: Int` field in the object layout, meaning that there's a higher chance that calling them will require an extra memory load.